pkg/Library/Core/arcs.js.map

```diff
@@ -1,7 +1,7 @@
 {
   "version": 3,
```

Comment:
question about all these files (arcs.js, arcs.js.map, etc) - are we submitting these every time they change? just doublechecking, as i'm never sure what to do about them.

Reply:
I'm currently treating them like 'arcsjs-packages', so yes, we are supposed to check in the built artifacts when we change the TS files.